src/assets/javascript/analytics/cookieBanner.js

```diff
@@ -1,5 +1,3 @@
-/* global document window */
```

Comment:
We currently have to do this in the DCMAW repo, but I believe the reason why is that there we use a TypeScript parser in our linting files. If we try to include the global declarations here we get an error that the global variables are already available in the browser